tools/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+add_library( freertos_plus_tcp_tools STATIC )
+
+target_sources( freertos_plus_tcp_tools
```

Comment:
Could this been included in the default FreeRTOS+TCP target?

Reply:
Yes that `could` be done - if that is the case then the tools/* code `should` be placed in the same directory as the freertos_plus_tcp code.

Typically a separate directory means a separate library.  Having multiple libraries here is not a big deal.  The up-side to leaving as-is in the tcp_utilities directory allows one to take this and combine this static library in their testing area so they have the same code monitoring the counts inside and outside the `Unit under test`.  Otherwise one would need to expose a private set of API's in both.

Assuming leaving here as-is - for reasons provided.